osquery/config/config.cpp

```diff
@@ -807,4 +824,20 @@ Status ConfigParserPlugin::setUp() {
   }
   return Status(0, "OK");
 }
+
+void ConfigRefreshRunner::start() {
+  while (!interrupted()) {
+    // Cool off and time wait the configured period.
+    // Apply this interruption initially as at t=0 the config was read.
+    pauseMilli(FLAGS_config_refresh * 1000);
+    // Since the pause occurs before the logic, we need to check for an
+    // interruption request.
+    if (interrupted()) {
+      return;
+    }
+
+    LOG(INFO) << "Refreshing configuration state";
```

Comment:
Thoughts on making this a `VLOG` as opposed to `LOG(INFO)`? Might be better if the refresh rate is relatively high to keep these messages out of view?